src/pages/[l1_l2]/[...slug].tsx

```diff
@@ -173,20 +174,29 @@ const Topic: NextPage<TopicProps> = ({
 export default Topic;
 
 interface IParams extends ParsedUrlQuery {
-    slug: string;
+    l1_l2: string;
+    slug: string[];
 } // Need this to avoid TS errors.
 
 export const getStaticPaths: GetStaticPaths = async () => {
-    const paths = products.map(({ slug }) => ({ params: { slug } }));
+    let paths: any[] = [];
+    taxonomyData.forEach((value: Taxonomy[], key: string) => {
+        const category = taxonomyToCategoryMapping[key];
+        paths = paths.concat(
+            value.map(({ slug }) => ({
+                params: { l1_l2: category, slug: slug.split('/') },
```

Comment:
params must have parameter matching folder structure hence l1-l2 and slug should be an array as opposed to string.
l1-l2 : "product" , slug : ['atlas'] - will generate path of mongdb.com/developer/product/atlas
l1-l2 : "product" , slug : ['atlas', 'changestreams'] - will generate path of mongdb.com/developer/product/atlas/changestreams